code/src/components/Footer.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import styled from "styled-components/macro";
+
+const StyledFooter = styled.p `
+  height: 30px;
+  background-color: cadetblue;
+  text-align: center;
+  font-size: 18px;
+  margin-top: auto;
+  padding-top: 2px;
+  font-weight: 600;
+`
+
+const Footer = () => <StyledFooter>© BumBumBurka & JenQuach</StyledFooter>
+
+export default Footer
```

Comment:
it looks pretty good to have styled component, clean and organized code. It's nice to keep everything together both styling and logic at the same place. Looking forward to learn more about styled components now :)